ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-periodics.yaml

```diff
@@ -13,17 +13,6 @@ periodics:
     ci-operator.openshift.io/prowgen-controlled: "true"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-assisted-service-master-e2e-metal-assisted-operator-disconnected-periodic
-  reporter_config:
```

Comment:
Looks like you accidentally deleted the assisted service slack messages :thinking: